apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,331 @@
+---
+slug: Getting Started MACI
+title: Getting Started with MACI
+description: Guide to use MACI
+authors:
+  name: Crisgarner
+  title: MACI team contributor
+  url: https://x.com/crisgarner
+  image_url: https://avatars.githubusercontent.com/u/578688?v=4
+tags: [voting, security, anonymity, roadmap, community, MACI, development]
+excerpt: "Get Started with MACI"
+---
+
+Hey folks, we are excited to start a series of tutorials on how to get started with all things MACI, this guide focuses on installing MACI and how to deploy the contracts to a testnet.
+
+## Understanding Roles
+
+MACI protocol has two main roles, User (Voter) and Coordinator. A simplified version would be to say that the coordinator is in charge of deploying the MACI smart contracts, inititalting the polls, allying the final results of a vote, and finalizing polls by publishing the final results on-chain. Usually the contract deployer is the coordinator, but this can be a separate role.
```

Comment:
you are mixing american vs british english version (z and s) e.g. finalizing vs finalising and later parameterised - stick to one

Reply:
Oh wow, I didn't know there was an American vs British version, will stick with american.